etc/provision.yml

```diff
@@ -0,0 +1,141 @@
+        admin: true
+      state: present
+
+  - name: Record Privileged User Token
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      username: "{{ admin_name }}"
+      password: "{{ admin_pass }}"
+      get:
+        path: "user/me"
+    register: admin
+
+  - name: Ensure Non-Privileged User
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      user:
+        login: "{{ user_name }}"
+        password: "{{ user_pass }}"
+        firstName: Girder
+        lastName: User
+        email: "user@girder.girder"
+        admin: false
+      state: present
+
+  - name: Ensure Data Sharing Group
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      group:
+        name: "dataset sharing"
+        description: "dataset sharing"
+        users:
+          - login: "{{ admin_name }}"
+            type: "admin"
+          - login: "{{ user_name }}"
+            type: "member"
+      state: present
+    register: dataSharingGroup
+
+  - name: Ensure Default Assetstore
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      assetstore:
+        name: "Local"
+        type: "filesystem"
+        current: true
+        root: "/local-filesystem"
+      state: present
+
+  - name: Activate Girder Plugins
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      plugins:
+        - autojoin
+        - bsve
+        - jobs
+        - worker
+        - database_assetstore
+        - girder_ktile
+        - minerva
+      state: present
+
+  - name: Restart Girder Server
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      put:
+        path: "system/restart"
+
+  - name: Wait for Girder Server
+    local_action:
+      module: wait_for host="{{ ghost }}" port="{{ gport }}" delay=10
+
+  - name: Update Girder System Settings
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      setting:
+        key: "{{ item.key }}"
+        value: "{{ item.value }}"
+      state: present
+
+    with_items:
+      - key: "worker.broker"
+        value: "{{ broker }}"
+
+      - key: "worker.backend"
+        value: "{{ broker }}"
+
+      - key: "worker.api_url"
+        value: "http://{{ ghost }}:{{ gport }}/api/v1"
+
+      - key: "autojoin"
+        value:
+          - groupId: "{{ dataSharingGroup['gc_return']['_id'] }}"
+            level: 0
+            pattern: "@"
+
+  - name: Restart Girder Server
+    girder:
+      host: "{{ ghost }}"
+      port: "{{ gport }}"
+      scheme: "http"
+      token: "{{ admin['token'] }}"
+      put:
+        path: "system/restart"
```

Comment:
The provisioning should also change the registration policy to "closed".  The geoviz plugin handles creating new users.

Reply:
Oh yes. I failed to mention this to Omar in previous conversations.